client/components/AbsoluteDate/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+
+/**
+ * Display absolute date in <time> element
+ *
+ * Usage:
+ * <AbsoluteDate date={historyItem._created} />
```

Comment:
We already have a `<Datetime/>` component that use the datetime format from the SD config. Did you notice it ? Is it incompatible with what you need ?